x-pack/plugins/observability/public/components/alerts_table/slo/get_slo_alerts_table_configuration.tsx

```diff
@@ -5,32 +5,32 @@
  * 2.0.
  */
 
-import type { GetRenderCellValue } from '@kbn/triggers-actions-ui-plugin/public';
-import { TIMESTAMP } from '@kbn/rule-data-utils';
 import { SortOrder } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { ALERT_DURATION } from '@kbn/rule-data-utils';
 import { AlertsTableConfigurationRegistry } from '@kbn/triggers-actions-ui-plugin/public/types';
 import { casesFeatureId, observabilityFeatureId } from '../../../../common';
-import { getRenderCellValue } from './render_cell_value';
+import { getRenderCellValue } from '../common/render_cell_value';
 import { columns } from './default_columns';
 import { useGetAlertFlyoutComponents } from '../../alerts_flyout/use_get_alert_flyout_components';
 import type { ObservabilityRuleTypeRegistry } from '../../../rules/create_observability_rule_type_registry';
 import type { ConfigSchema } from '../../../plugin';
-import type { TopAlert } from '../../../typings/alerts';
-import { SLO_ALERTS_TABLE_CONFID } from '../../../embeddable/slo/constants';
+import { SLO_ALERTS_TABLE_CONFIG_ID } from '../../../embeddable/slo/constants';
 
 export const getSloAlertsTableConfiguration = (
   observabilityRuleTypeRegistry: ObservabilityRuleTypeRegistry,
   config: ConfigSchema
 ): AlertsTableConfigurationRegistry => ({
-  id: SLO_ALERTS_TABLE_CONFID,
+  id: SLO_ALERTS_TABLE_CONFIG_ID,
   cases: { featureId: casesFeatureId, owner: [observabilityFeatureId] },
   columns,
-  getRenderCellValue: (({ setFlyoutAlert }: { setFlyoutAlert: (data: TopAlert) => void }) => {
-    return getRenderCellValue({ observabilityRuleTypeRegistry, setFlyoutAlert });
-  }) as unknown as GetRenderCellValue,
+  getRenderCellValue: ({ setFlyoutAlert }) =>
+    getRenderCellValue({
+      observabilityRuleTypeRegistry,
+      setFlyoutAlert,
+    }),
   sort: [
     {
-      [TIMESTAMP]: {
+      [ALERT_DURATION]: {
```

Comment:
@mgiota, @umbopepato found the cause of the forever loading issue and it was due to sorting based on a field that we didn't have in the default columns.

Great finding @umbopepato 💪🏻